ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,38 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# bach_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.bach_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
`layer_norm` does not have the same arguments as `batch_norm`. You'll need to write another composite strategy to generate the data for this function. You should generate a value for each of the arguments in your `st.composite` strategy - you can take a look at the test of `ivy.batch_norm` to get an idea of the same

Reply:
Ok, let me work on trying to write another composite strategy and I will get back you soon...